### ISSUE TITLE
Double Check Unique Expired Service Accounts for output potentially being skipped

### ISSUE THREAD

**Issue by infamousjoeg** [infamousjoeg/DNAmicAnalysis#101]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by infamousjoeg** [comment]
@kazekan I get the following output in Excel when running my test:

```
Oldest Non-Compliant Service: Mike
Max Password Age: 546 days (1.5 years)
Total Avg Password Age: 935 / 2 = 467.50 days (1.3 years)
Total Percent Non-Compliant: 2 / 6 = 33.33%
--
Unique Expired Services
--
Mike
svc_sql
```

Is this the metric you wanted to double check?

**Comment by infamousjoeg** [comment]
This was fixed by #117
